src/main/java/com/theocean/fundering/domain/celebrity/controller/FollowController.java

```diff
@@ -0,0 +1,46 @@
+package com.theocean.fundering.domain.celebrity.controller;
+
+import com.theocean.fundering.domain.celebrity.service.FollowService;
+import com.theocean.fundering.global.jwt.userInfo.CustomUserDetails;
+import com.theocean.fundering.global.utils.ApiResult;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+@Tag(name = "FOLLOW", description = "celeb 팔로우 관련 API")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api")
+public class FollowController {
+    private final FollowService followService;
+
+    @Operation(summary = "셀럽 팔로우", description = "셀럽의 아이디를 기반으로 팔로우한다.")
+    @PreAuthorize("hasRole('ROLE_USER')")
+    @PostMapping("/celebs/{celebId}/follow")
+    @ResponseStatus(HttpStatus.OK)
+    public ApiResult<?> followCelebs(
+            @Parameter(description = "셀럽의 PK") @PathVariable final Long celebId,
+            @AuthenticationPrincipal final CustomUserDetails userDetails
+    ) {
+        followService.followCelebs(celebId, userDetails.getId());
+        return ApiResult.success(null);
```

Comment:
```suggestion
        return ApiResult.success();
```
메서드 오버로딩을 통해 자동으로 payload에 null을 넣어주면 어떨까요?